examples/coap-server.c

```diff
@@ -2434,6 +2454,84 @@ cmdline_read_user(char *arg, unsigned char **buf, size_t maxlen) {
 }
 #endif /* SERVER_CAN_PROXY */
 
+static FILE *oscore_seq_num_fp = NULL;
```

Comment:
(the same comments as for coap-client.c)

Reply:
Updated with the exception to `buf `in get_`oscore_conf()`.